src/models/UserModel.js

```diff
@@ -0,0 +1,29 @@
+import db from '../apiCalls';
+
+class UserModel {
+  constructor() {
+    this.allUsers = this.getAllCustomers();
+    this.currentUser = {};
+  }
+
+  getAllCustomers() {
+    return db.getAllCustomers()
+    .then(data => return data);
+  }
+
+  getSingleCustomer(userId) {
+    this.currentUser = db.getSingleCustomer(userId)
```

Comment:
If this is returning an object, consider declaring `this.currentUser` without assignment to an empty object on line 6.